server/src/main/java/org/spine3/server/storage/AbstractStorage.java

```diff
@@ -58,6 +58,7 @@
      * @param id the ID for the record
      * @param record a record to store
      * @throws IllegalStateException if the storage was closed before
+     * @throws NullPointerException if the ID or record is {@code null}
```

Comment:
We use `@ParametersAreNonnullByDefault` on the level of the package, which assumes that everything, which is not `@Nullable` is going to cause NPE when passed.